core/src/main/res/values/strings.xml

```diff
@@ -45,6 +45,7 @@
   <string name="server_started_message" tools:keep="@string/server_started_message">Enter this ip address into your browser to access the server %s</string>
   <string name="share_host_address">Share URL via other applications</string>
   <string name="error_file_not_found">Error: The selected ZIM file could not be found.</string>
+  <string name="unable_to_read_zim_file" translatable="false">Unable to read this zim file!</string>
```

Comment:
Why this can't be translatable?

Reply:
do you want to add in all strings file using online converter ?

Reply:
No! Transaltewiki will do that for us. We just need to add the English version of the string. My question was why are you adding this `translatable="false"` parameter?

Reply:
ok i just add as safe side so that if in any file this is missing app will not crashed? do you want to remove it  ?

Reply:
Yes. Where it needs to be removed, remove it.

Reply:
ok

Reply:
done